api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceDao.java

```diff
@@ -103,18 +103,12 @@ Optional<DbWorkspace> findFirstByWorkspaceNamespaceAndActiveStatusOrderByLastMod
   Optional<DbWorkspace> findFirstByGoogleProjectAndActiveStatusOrderByLastModifiedTimeDesc(
       String googleProject, short activeStatus);
 
-  List<DbWorkspace> findAllByBillingMigrationStatus(Short billingMigrationStatus);
-
   DbWorkspace findDbWorkspaceByWorkspaceId(long workspaceId);
 
   Set<DbWorkspace> findAllByCreator(DbUser user);
 
   List<DbWorkspace> findAllByNeedsResearchPurposeReviewPrompt(short researchPurposeReviewed);
 
-  default List<DbWorkspace> findAllWithBillingMigrationNewStatus() {
```

Comment:
Removed in #6882

Reply:
and the final query relating to the field was removed in a later iteration of the same: #6922